java/src/jmri/jmrit/roster/Roster.java

```diff
@@ -321,6 +321,23 @@ public RosterEntry getEntryForId(String id) {
         return null;
     }
 
+     /**
+     * Return a list of RosterEntry which have a 
+     * particular DCC address.
+     *
+     * @param a 
+     * @return an ArrayList of matching entries
+     */
+    public List<RosterEntry> getEntriesByDccAddress(String a) {
+        List<RosterEntry> list = new ArrayList<>(); 
```

Comment:
You really want to return the results of matchingList(...) with all null arguments except for the dccAddress argument instead of looping through _list yourself.